packages/web-components/src/recommendations/content/content-viewed-after-viewing-multiple-products.ts

```diff
@@ -0,0 +1,64 @@
+import { ContentRecommendationResponse, ContentsViewedAfterViewingMultipleProductsBuilder, ContentsViewedAfterViewingMultipleProductsRequest } from '@relewise/client';
+import { ContentRecommendationBase } from './content-recommendation-base';
+import { getRecommender } from '../recommender';
+import { getRelewiseUIOptions } from '../../helpers/relewiseUIOptions';
+import { getContentRecommendationBuilderWithDefaults } from '../../builders/contentRecommendationBuilder';
+
+export class ContentViewedAfterViewingMultipleProducts extends ContentRecommendationBase {
+
+    private productAndVariantIds: { productId: string; variantId?: string }[] = [];
```

Comment:
`{ productId: string; variantId?: string }` -> I actually think you can just use `ProductAndVariantId` type from the SDK here.